src/Language/PureScript/Publish/Utils.hs

```diff
@@ -16,14 +16,20 @@ import qualified System.FilePath.Glob as Glob
 globRelative :: Glob.Pattern -> IO [FilePath]
 globRelative pat = do
   currentDir <- getCurrentDirectory
-  filesAbsolute <- Glob.globDir1 pat currentDir
+  glob pat currentDir
+
+-- | Glob relative to the specified directory, and produce relative pathnames.

```

Comment:
Well, I want to find files in `target`, but then test for a prefix which begins with the current directory. Maybe I should add that as a comment?